website/events/models/event_registration.py

```diff
@@ -33,7 +33,6 @@ class EventRegistration(models.Model):
         models.CASCADE,
         blank=True,
         null=True,
-        limit_choices_to=registration_member_choices_limit,
```

Comment:
I did not remove the relevant filter, because it is still used in makefixtures. I'd prefer fixtures to not do weird things here (such as register for events without a membership while this is not allowed for the event in question).